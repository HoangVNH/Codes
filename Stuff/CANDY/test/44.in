10
80
XDVDVDVVXDVVDXDXXDXVVXXDXVDXXDDXVXXDVVVDVXDDDDVXXVDDDDXVDVVDXXDXVXVVDXDDDDXXDVDV
94
DDVXDXDDDXXDXVXVVXVVDDDVVXDXXDDDXXDVXXDXVVVVVDDXVVDXXDDXVXVDVXDXDDDVDVXDDXXVXDVVVVXDVXVDVVXXDX
74
DXXVVVXVDXVVVXDVDDDXDDVVXVXVVVVDVXDXVDVXDVXVVVDDVDVXDDXXXXVDDXVVXVXDVDVVVX
84
XXDXVVDDDXDVXVXVXDXXDDVVDXXVVVDVVXXVVXVXVDXDVDVXXDXVXXDVDDVXDDXVXXXVVVDVVDXDDVDVVXVD
100
DXXXDVXVXVVVDXXDXDXDDDXXVXDDVDXVDDDDXDDXXDDVDXVVXDVXVDDVXDXDDXXVDXVVDDXXXVXVDVVXVXVXXVXDVVVDVXXXDVVV
84
DVXDVVDXXXXXDDVDVVDXXXXXVXXXVXXDXVDVXVVDVXVDVDXVXVVDDVXDDDXVXXVVDVVDXDDDXVVVXXVVXXXD
92
XVVVDVXDDDXVDXDXXDVXXXDVDXDXXVDDXVDXDDXXVXVXXDDDXDVVDDDVXDXXDVDXDDXDVVDVXXXDVDVVXVXVXXDDVDXD
58
DDVXDXXVXVDDXDXVDDXXVDXDXXDVVXXVVXXDDDDVXDDXXXXVDVDDVXVDVX
55
VXXDXXDXXXXVDXVDXXXVVDVDXDXXVXVDVXDVDXXXVXDVXDDXXVXXVXD
84
VDVDDVXDVVVVDVVXDDVVDXXXVVDDVVDDXVXXXVDDVXDXDVVXXXXVVXDVDVXDVXVDVVXDXXDVXXXDDVXVVVDD
