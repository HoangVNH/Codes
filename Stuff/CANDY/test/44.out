79
93
73
83
99
83
91
57
54
83
